cmake/FindMOAB.cmake

```diff
@@ -49,7 +49,7 @@ else (MOAB_LIBRARIES AND MOAB_INCLUDE_DIRS)
   # handle the QUIETLY and REQUIRED arguments and set MOAB_FOUND to TRUE
   # if all listed variables are TRUE
   find_package_handle_standard_args(MOAB  DEFAULT_MSG
-                                    MOAB_LIBRARY MOAB_INCLUDE_DIR)
+      MOAB_LIBRARY MOAB_INCLUDE_DIRS)
```

Comment:
This looks suspicious- should it be `MOAB_LIBRARIES` as well?

Reply:
We have already changed to MOAB_INCLUDE_DIRS through PR #1104

Reply:
@bam241, You can proceed to close the issue if you want. As it is already changed in latest PyNE.